### DISCUSSION TITLE
11/30/2022 Shipments

### DISCUSSION THREAD

**Discussion by arda858** [joeycastillo/Sensor-Watch#148] [General]
Hi all, I discovered this project last week and ordered a board and temp sensor right away! Looking forward to receving it. I have my watch and code ready!

Is anyone else ordered? How long does it usually take to ship?

**Comment by joeycastillo** [comment]
If you ordered today, you should expect it by late December or early January, hopefully! I just shipped 500 boards to Mouser, Crowd Supply’s distribution partner, and there are 200 more headed there next week. This is enough to fulfill all the pre-orders.

**Reply by arda858** [reply]
Thanks for the reply Joey! I ordered it soon as I saw it on November 24th :) Brilliant project. I can't wait to build it and wear it as my daily.
